front-end/src/app/shared/services/login.service.ts

```diff
@@ -66,12 +66,10 @@ export class LoginService {
     if (this.userLoginData && this.userLoginData.token) { // Non-login.gov auth
       this.store.dispatch(userLoggedOutAction());
     } else {
-      this.apiService.postAbsoluteUrl(`${environment.loginDotGovLogoutUrl}`, null).pipe(
-        tap(() => {
-          this.clearUserLoggedInCookies();
-          this.store.dispatch(userLoggedOutAction());
-        })
-      ).subscribe(() => undefined);
+      this.cookieService.delete('csrftoken');
+      window.location.href = environment.loginDotGovLogoutUrl || "";
```

Comment:
This seems to be causing the unit tests to fail.  Maybe there's a way to turn this off when it's running unit tests